.github/workflows/swift-toolchain.yml

```diff
@@ -200,6 +212,9 @@ jobs:
           echo windows_build_runner=${{ vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
           echo compilers_build_runner=${{ vars.COMPILERS_BUILD_RUNNER || vars.WINDOWS_BUILD_RUNNER || 'windows-latest' }} >> ${GITHUB_OUTPUT}
 
+          echo ANDROID_API_LEVEL=${{ env.ANDROID_API_LEVEL }} >> ${GITHUB_OUTPUT}
+          echo WINDOWS_VISUAL_STUDIO_PATH="C:\Program Files\Microsoft Visual Studio\2022\Enterprise" >> ${GITHUB_OUTPUT}
```

Comment:
Please don't do this. We should query this using `vswhere`.

Reply:
I have Community version of VS where this wouldn't work?

Reply:
Updated to use vswhere